settings.py

```diff
@@ -509,11 +504,11 @@
 GET_URL_BASE = 'dimagi.utils.web.get_url_base'
 
 # celery
-BROKER_URL = 'redis://localhost:6379/0'
+CELERY_BROKER_URL = 'redis://localhost:6379/0'
```

Comment:
@dimagi/team-commcare-hq  Heads up that we're now running celery 4.  To run celery locally will require updating localsettings with the variable name changes here, and with the changes in [localsettings.py](https://github.com/dimagi/commcare-hq/pull/23650/files#diff-75c03275bdad3fd077a71ee5dfd90065)

BROKER_URL -> CELERY_BROKER_URL
CELERY_ALWAYS_EAGER -> CELERY_TASK_ALWAYS_EAGER
CELERY_ANNOTATIONS -> CELERY_TASK_ANNOTATIONS
CELERYD_TASK_SOFT_TIME_LIMIT -> CELERY_TASK_SOFT_TIME_LIMIT